<launch>
	<include file="$(find turtlebot_follower)/launch/includes/_velocity_smoother.launch"/>
  	<node pkg="dfDrone" type="nodelet" name="turtlebot_follower"
        	args="load turtlebot_follower/TurtlebotFollower camera/camera_nodelet_manager">
    		<remap from="turtlebot_follower/cmd_vel" to="/cmd_vel_mux/input/navi_raw"/>
    		<remap from="turtlebot_follower/depth/points" to="camera/depth/points"/>
    		<param name="enabled" value="true" />
    		<param name="x_scale" value="7.0" />
    		<param name="z_scale" value="2.0" />
    		<param name="min_x" value="-0.35" />
    		<param name="max_x" value="0.35" />
    		<param name="min_y" value="0.1" />
    		<param name="max_y" value="0.5" />
    		<param name="max_z" value="1.2" />
    		<param name="goal_z" value="0.6" />
	</node>
	<node pkg="dfDrone" name="centerNode" required="true" type="centerDrone" output="screen"></node>

	<node pkg="dfDrone" name="startNode" required="true" type="startDrone" output="screen"></node>
</launch>
